src/videojs-hls.js

```diff
@@ -829,7 +846,14 @@ videojs.HlsHandler.prototype.fillBuffer = function(mediaIndex) {
     tech = this.tech_,
     currentTime = tech.currentTime(),
     hasBufferedContent = (this.tech_.buffered().length !== 0),
-    currentBuffered = this.findBufferedRange_(),
+
+    // !!The order of the next two assignments is important!!
+    // `currentTime` must be equal-to or greater-than the start of the
+    // buffered range. Flash executes out-of-process so, every value can
+    // change behind the scenes from line-to-line. By reading `currentTime`
+    // after `buffered`, we ensure that it is always a current or later
+    // value during playback.
+    currentBuffered = videojs.Hls.Ranges.findRange_(tech.buffered(), tech.currentTime()),
```

Comment:
I think it still does. In this case, the order the arguments are evaluated as they are passed to findRange_() is actually relevant.